overlay.py

```diff
@@ -45,11 +45,20 @@ def __init__(self, bike, width=1280, height=740, bg: str = None):
         self.data = DataFactory.create(bike_version)
         self.device = configs["device"]
 
+        # MQTT client options
         self.client = mqtt.Client()
         self.client.on_connect = self._on_connect
         self.client.on_disconnect = self.on_disconnect
         self.client.on_log = self.on_log
 
+        # Set the camera status to offline if connection breaks
+        is_online_topic = (
```

Comment:
Just checked the commit messages in #41. If you are changing `send_camera_is_online` to `send_video_status`, could this also be changed to `video_status_topic`? 

Would like to get @haotongwang's opinion first since it was his change, idm either option